prisma/schema.prisma

```diff
@@ -1,25 +1,29 @@
-// This is your Prisma schema file,
-// learn more about it in the docs: https://pris.ly/d/prisma-schema
-
-generator client {
-    provider = "prisma-client-js"
-}
-
-datasource db {
-    provider = "postgresql"
-    url      = env("DATABASE_URL")
-}
-
-model GuildConfig {
-    guildId     String       @id
-    roleId      String?
-    channelId   String
-    wordpress   String
-    lastChecked LastChecked?
-}
-
-model LastChecked {
-    last    DateTime
-    guildId String      @id
-    guild   GuildConfig @relation(fields: [guildId], references: [guildId])
-}
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+    provider = "prisma-client-js"
+}
+
+datasource db {
+    provider = "postgresql"
+    url      = env("DATABASE_URL")
+}
+
+model GuildConfig {
+    guildId     String       @id
+    roleId      String?
+    channelId   String
+    wordpress   String
+    lastChecked LastChecked?
+}
+
+model LastChecked {
+    last    DateTime
+    guildId String      @id
+    guild   GuildConfig @relation(fields: [guildId], references: [guildId])
+}
+
+model PostedId {
+    id    Int           @id
```

Comment:
we should add a `wordpress String` (the enable this bot to utilize multiple Wordpress instances at once.

Reply:
`wordpress String @id` then the id won't be the ID since it wouldn't be unique anymore

Reply:
then it'd only do one of the servers since they have the same Wordpress url

Reply:
wait no nvm